### PR TITLE
fix: update cation --- showjim

### PR DESCRIPTION
**Pull request by showjim**
this is a sample fix PR

### CODE REVIEW

File: _slides/10-showjim.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+learn to use git
```
Comment (yehjovi) [inline _slides/10-showjim.md:7]: HI

**Review by yehjovi — COMMENTED** [review]
HI

File: _slides/10-showjim.md (hunk 1 above)
Comment (showjim) [reply _slides/10-showjim.md:7]: Hi

File: _slides/10-showjim.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
```
Comment (rogermow) [inline _slides/10-showjim.md:6]: Looks good.

**Review by rogermow — APPROVED** [review]

**Pull request merged**